src/auth/mutations/signup.ts

```diff
@@ -34,5 +35,26 @@ export default resolver.pipe(resolver.zod(Signup), async ({ email, password, use
     data: widgets,
   })
 
+  const msg = {
```

Comment:
just an idea: should we store the messages separately at one place?

Reply:
Yes, was going to work on that later